controllers/RecruiterConroller.js

```diff
@@ -0,0 +1,111 @@
+const { body, validationResult, sanitizeBody } = require("express-validator");
+const apiResponse = require("../helpers/apiResponse");
+const { constants } = require("../helpers/constants");
+const auth = require("../middlewares/jwt");
+const RecruiterModel = require("../models/RecruiterModel");
+const UserModel = require("../models/UserModel");
+
+exports.Recruiter = [
+  auth,
+  body("companyName", "company Name must not be empty.").notEmpty(),
+  body("userRole", "user Role must not be empty.").notEmpty(),
+  body("mobileNo", "mobileNo must be 10 digits.").isLength({
+    min: 10,
+    max: 10,
+  }),
+  body("locations", "locations must not be empty.").notEmpty().isArray(),
+  body("website", "website must not be empty.").notEmpty(),
+  body("foundationYear", "foundation Year must not be empty.").isNumeric(),
+  body("noOfEmployees", "no Of Employees must not be empty.").isNumeric(),
+  body("aboutCompany", "about must not be empty.").notEmpty(),
+  body("companyPhotos", "company Photos must not be empty.").isArray({
+    max: 5,
+  }),
+  body("linkedinProfile", "linkedin Profile must not be empty.").notEmpty(),
+  body("twitterProfile", "twitter Profile must not be empty.").notEmpty(),
+  body("facebookProfile", "facebook Profile must not be empty.").notEmpty(),
+  body("companyLogo", "companyLogo must be empty.").notEmpty(),
+  (req, res) => {
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 1 tab but found 2 spaces.](https://app.codacy.com/gh/n4alphapesto/hirebook-backend/pullRequest?prid=7771833)
```suggestion
	(req, res) => {
```